.appveyor.yml

```diff
@@ -27,17 +27,17 @@ install:
       Write-Host "Installed" -ForegroundColor Green
   - ps: |
       Write-Host "Installing GStreamer..." -ForegroundColor Cyan
-      $msiPath = "$($env:USERPROFILE)\gstreamer-1.0-x86_64-1.14.4.msi"
+      $msiPath = "$($env:USERPROFILE)\gstreamer-1.0-msvc-x86_64-1.18.1.msi"
       Write-Host "Downloading..."
-      (New-Object Net.WebClient).DownloadFile('https://s3-us-west-2.amazonaws.com/qgroundcontrol/dependencies/gstreamer-1.0-x86_64-1.14.4.msi', $msiPath)
+      (New-Object Net.WebClient).DownloadFile('https://gstreamer.freedesktop.org/data/pkg/windows/1.18.1/msvc/gstreamer-1.0-msvc-x86_64-1.18.1.msi', $msiPath)
```

Comment:
@patrickelectric will you put this on s3?

Reply:
1. https://s3.amazonaws.com/downloads.bluerobotics.com/QGC/dependencies/gstreamer-1.0-devel-msvc-x86_64-1.18.1.msi
2. https://s3.amazonaws.com/downloads.bluerobotics.com/QGC/dependencies/gstreamer-1.0-msvc-x86_64-1.18.1.msi